azext_iot/iothub/params.py

```diff
@@ -0,0 +1,32 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+def load_iothub_arguments(self, _):
+    """
+    Load CLI Args for Knack parser
+    """
+    with self.argument_context("iot pnp twin") as context:
+        context.argument(
+            "command_name",
+            options_list=["--command-name", "--cn"],
+            help="Digital twin command name.",
+        )
+        context.argument(
+            "component_path",
+            options_list=["--component-path"],
+            help="Digital twin component path.",
+        )
+        context.argument(
+            "json_patch",
+            options_list=["--json-patch", "--patch"],
+            help="An update specification described by JSON-patch. "
+            "Operations are limited to add, replace and remove. Provide file path or inline JSON.",
+        )
+        context.argument(
+            "payload",
+            options_list=["--payload"],
```

Comment:
i think the pattern elsewhere is to use -k or --content

Reply:
For the IoT hub invoke device method command the usage there is `--method-payload/--mp` ...for IoT Hub configurations (like edge deployments or device management) and the `az iot central app device run-command`  `-k` or `--content` is used. I think `payload` fits the story we want to tell our users -- maybe in a future iteration `--payload` can be a short option for `--method-payload`.